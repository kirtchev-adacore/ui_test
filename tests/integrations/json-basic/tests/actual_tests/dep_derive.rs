//@run

#[macro_use]
extern crate serde_derive;

fn main() {}
