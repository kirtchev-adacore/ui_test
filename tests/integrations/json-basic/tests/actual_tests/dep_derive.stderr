warning: unused `#[macro_use]` import
 --> tests/actual_tests/dep_derive.rs:3:1
  |
3 | #[macro_use]
  | ^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: 1 warning emitted

